java/code/src/com/suse/manager/webui/menu/MenuTree.java

```diff
@@ -511,18 +513,29 @@ private MenuItem getAboutNode(String docsLocale) {
             .addChild(new MenuItem("Search").withPrimaryUrl("/rhn/help/Search.do"));
     }
 
-    private MenuItem getExternalLinksNode() {
+    private MenuItem getExternalLinksNode(String docsLocale) {
         return new MenuItem("External Links").withIcon("fa-link")
             .addChild(new MenuItem("header.jsp.knowledgebase")
-                    .withPrimaryUrl("https://www.suse.com/support/kb/product.php?id=SUSE_Manager")
+                    .withPrimaryUrl("https://www.suse.com/support/kb/?id=SUSE+Manager")
                     .withTarget("_blank"))
             .addChild(new MenuItem("header.jsp.documentation")
                     .withPrimaryUrl(ConfigDefaults.get().isUyuni() ?
-                            "https://www.uyuni-project.org/uyuni-docs/uyuni/index.html" :
-                            "https://documentation.suse.com/suma/")
+                            "https://www.uyuni-project.org/uyuni-docs/" + docsLocale + "/uyuni/index.html" :
+                            "https://documentation.suse.com/suma/" + getMajorMinorProductVersion() + "/")
                     .withTarget("_blank"));
     }
 
+    /**
+     * Get the `MAJOR.MINOR` part of the product version string
+     */
+    private String getMajorMinorProductVersion() {
```

Comment:
I couldn't find a way either, but I'm not very savvy at this side of the codebase. Ping @mcalmer just in case, and also since you get pinged so rarely.

Reply:
I think this is the best way we have. As long as the regex matching works and with Uyuni it does not break I am fine.